samcli/lib/providers/exceptions.py

```diff
@@ -84,3 +84,12 @@ def resource_identifier(self) -> "ResourceIdentifier":
     @property
     def property_name(self) -> str:
         return self._property_name
+
+
+class MissingTemplateFile(UserException):
```

Comment:
Similar to the above comment, we already have a `TemplateNotFoundException` we can re-use.